src/Containers/Impl/IContainer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.R.Containers {
+    public interface IContainer {
+        string Id { get; set; }
```

Comment:
That is container technology specific. I will add it to the docker specific container implementation.

---
In reply to: [128899647](https://github.com/Microsoft/RTVS/pull/3734#discussion_r128899647) [](ancestors = 128899647)